lib/web_ui/dev/test_runner.dart

```diff
@@ -205,6 +217,11 @@ class TestCommand extends Command<bool> with ArgUtils {
   /// Whether [browser] is set to "chrome".
   bool get isChrome => browser == 'chrome';
 
+  /// Use system flutter instead of cloning the repository.
```

Comment:
Let's also mention here that we use `PATH` to locate the Flutter repo.

Reply:
done.